### PR TITLE
Fix the second passage to alien base's navmesh

### PR DESCRIPTION
**Pull request by necessarily-equal**
Fix part of #5, the first problem in that issue.

After: (and with a map compilation in netradiant, `/navgen all` and `/map_restart`)

Screenshot from 2022-10-30 04-38-41

We can see the navmesh is continuous

### CODE REVIEW

File: maps/perseus.map [hunk 1]
```diff
@@ -63848,11 +63840,11 @@ shared_ex/base1a
 ( 64 1360 464 ) ( -64 1368 464 ) ( -64 1360 464 ) common/caulk 0 0 0 0.25 0.25 0 0 0
 }
 {
-( 904 -192 128 ) ( 904 -224 128 ) ( 872 -192 128 ) perseus_ex/base1a_nonsolid 256 192 0 -0.25 0.25 0 0 0
```
Comment (illwieckz) [inline maps/perseus.map:63851]: I recommend you to duplicate `textures/perseus_ex/base1a_nonsolid` as `textures/perseus_ex/base1a` in `scripts/perseus_ex.shader` and remove nonsolid parm, and use `perseus_ex/base1a` instead.

Comment (necessarily-equal) [reply maps/perseus.map:63851]: tbh what was there previously looks like a mistake. There were two brush with similar looks nut differently-sized features so I just replaced that to use one brush with a floor grid texture from another part of the map.

It really just looked like the texture choice wasn't intentional

Comment (illwieckz) [reply maps/perseus.map:63851]: okok

**Review by illwieckz — APPROVED** [review]
LGTM

**Comment by illwieckz** [comment]
The surface was translucent and then the visibility was leaking, breaking map compilation.

I fixed it in f34a116a175b35812ea1d52e4dacedcd33ec776f

**Comment by necessarily-equal** [comment]
Oh derp, thanks!

Note to self: testing maps using novis isn't enough

**Comment by illwieckz** [comment]
We have a “leak test” build profile in NetRadiant build menu. 😉️

**Pull request merged**